static/elements/chromedash-roadmap-milestone-card.js

```diff
@@ -1,6 +1,6 @@
 import {LitElement, html, nothing} from 'lit';
 import {ROADMAP_MILESTONE_CARD_CSS} from
-  '../sass/elements/chromedash-roadmap-milestone-card-css.js';
+'../sass/elements/chromedash-roadmap-milestone-card-css.js';
```

Comment:
Usually we use indentation to show that the line is being continued from the previous line.  So, let's keep this indent.

Reply:
I think this was automatic formatting - I'll remove this.